controllers/NgoController.js

```diff
@@ -57,17 +62,39 @@ exports.export = [
   },
 ];
 
-exports.status = [
-  function (req, res) {
-    const query = [{ $group: { _id: "$status", status: { $sum: 1 } } }];
-    handleStatus(res, query, Ngo);
-  },
-];
-
 // Verify the NGO Registration details
 exports.updateStatus = [
   auth,
   function (req, res) {
+    // Todo: check email verification and create user
     handleStatusUpdate(req, res, Ngo);
   },
 ];
+
+exports.verifyEmail = [
+  asyncH(async (req, res) => {
+    let query = req.body
```

Comment:
I recommend picking only the keys needed from body in to query. 
const query = {
 data : req.body.data
}